js/table/actionBlock.ts

```diff
@@ -119,6 +119,7 @@ export class ActionBlock {
     public button3Visible = ko.observable(false);
     public needBB: KnockoutObservable<boolean>;
     public myPlayer: KnockoutObservable<TablePlaceModel>;
+    public isPortraitMode = ko.observable(appConfig.ui.usePortraitModeOnly);
```

Comment:
Could you change from `ko.observable` to `ko.computed` to indicate that that value always readonly.